src/app/qan/profile-table/profile-table.component.scss

```diff
@@ -62,7 +62,7 @@
 
 .table-wrapper--inner {
   --row-heigth: 40px;
-  --first-c-width: 30px;
+  --first-c-width: 40px;
```

Comment:
This might not work for cases with huge variety of queries. For example 1000 will cause same issue.
Is it possible to use `em` or similar instead of px?

Reply:
Yes, it is possible to use em, but it does not help in case with 1000, because em calculates regarding parent container font size and does not depend from child container width. For using adaptive measurements we need to remade structure and use grid instead of tables, but with our table behaviour in causes some bugs. In future we can do this, but wee need good browser support of css sub-grid https://caniuse.com/#feat=css-subgrid

Reply:
Than ok, for now.